src/components/select/Select.js

```diff
@@ -498,6 +498,22 @@ export class SelectComponent extends BaseComponent {
     }
   }
 
+  getView(data) {
+    if (!data) {
+      return '';
+    }
+    if (data && !this.useTemplate) {
+      return this.t(data.label || data);
+    }
+    if (typeof data === 'string') {
+      return this.t(data);
+    }
+    if (this.component.template) {
+      return this.interpolate(this.component.template, {item: data})
+    }
+    return data.label || this.component.placeholder;
```

Comment:
I cannot think of a good use case where you would want to return the "label" of the component as its "data" view. If the data is empty, then just return empty string. Actually, why can you not just do this?...

```js
getView(data) {
  return this.itemTemplate(data);
}
```

Reply:
I tried that but the "replace" method isn't working